app/models/stat.rb

```diff
@@ -0,0 +1,2 @@
+class Stat < ApplicationRecord
```

Comment:
Are we calling this a stat because OL return codes are only used for DfE analytics?

Reply:
Ticket says it needs to be a new thing, so I added this. I'm thinking `Stat` could be a generic thing and this may a useful and quick thing just to stick in data points that can consumed in big query?

Reply:
Good thinking - as a suggestion we could have a type column and then make the value column a generic name so it’s more easily reusable.

Reply:
done